pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -793,6 +816,329 @@ func TestGetPipelineFunc_VerifyError(t *testing.T) {
 	}
 }
 
+func TestGetPipelineFunc_V1Pipeline_VerifyNoError(t *testing.T) {
+	ctx := context.Background()
+	signer, _, k8sclient, vps := test.SetupVerificationPolicies(t)
+	tektonclient := fake.NewSimpleClientset()
+
+	v1beta1UnsignedPipeline := &v1beta1.Pipeline{
```

Comment:
does it make sense to have also testing with v1UnsignedPipeline/ v1Signed? 

this also felt like something we might be able to use as a helper function for the taskRef verification PR as well?